Sources/SyrupCore/GraphQL/OperationVisitor.swift

```diff
@@ -161,7 +168,10 @@ final class OperationVisitor: GraphQLBaseVisitor {
 	
 	override func visitSelectionSet(selectionSet: [SwiftGraphQLParser.Selection]) {
 		currentOperationContents += " {\n"
-		currentOperationContents += "__typename\n"
+		if currentOperationType != .subscription || !isRoot {
+			currentOperationContents += "__typename\n"
+		}
+		isRoot = false
```

Comment:
@jaredh didn't have the patience to figure out how to add `__typename` properly as a field